### PR TITLE
DDF-1305: Address all high severity CLM vulnerabilities

### PR DESCRIPTION
**Pull request by tbatieConnexta**
- Changed version that contained embedded jars with security vulnerabilities.

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf/66)

### CODE REVIEW

**Comment by tbatieConnexta** [comment]
@vinamartin @figliold @lessarderic

File: distribution/webconsole-branding/pom.xml [hunk 1]
```diff
@@ -40,7 +40,7 @@
         <extensions>true</extensions>
         <configuration>
           <instructions>
-            <Fragment-Host>org.apache.karaf.webconsole.console;bundle-version="[2,3)"</Fragment-Host>
+            <Fragment-Host>org.apache.karaf.webconsole.console;bundle-version="2.4.1"</Fragment-Host>
```
Comment (lessarderic) [inline distribution/webconsole-branding/pom.xml]: Try using [2.4.1,3) here instead.

Comment (tbatieConnexta) [inline distribution/webconsole-branding/pom.xml]: Fixed

**Comment by lessarderic** [comment]
:+1:

**Comment by figliold** [comment]
:+1: 

---

Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/66)

**Comment by figliold** [comment]
Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/66)

File: distribution/webconsole-branding/pom.xml [hunk 2]
```diff
@@ -40,7 +40,7 @@
         <extensions>true</extensions>
         <configuration>
           <instructions>
-            <Fragment-Host>org.apache.karaf.webconsole.console;bundle-version="[2,3)"</Fragment-Host>
+            <Fragment-Host>org.apache.karaf.webconsole.console;bundle-version="[2.4.2,3)"</Fragment-Host>
```
Comment (stustison) [inline distribution/webconsole-branding/pom.xml]: This should be 2.4.1

**Comment by shaundmorris** [comment]
As an FYI @clockard is working on https://github.com/codice/ddf/pull/74. There is an upgrade to Karaf 2.4.3 so some of this may be able to get handled on that PR

**Comment by shaundmorris** [comment]
Verified build, passing tests. Changes look good.

**Pull request merged**